src/node/config/scripts/generateNodeIndex.mjs

```diff
@@ -0,0 +1,135 @@
+/**
+ * This script generates the exports functionality for the node API.
+ *
+ * it exports chef as default, but all the wrapped operations as
+ * other top level exports.
+ *
+ * @author d98762656 [d98762625@gmail.com]
+ * @copyright Crown Copyright 2018
+ * @license Apache-2.0
+ */
+
+/*eslint no-console: 0 */
+
+import fs from "fs";
+import path from "path";
+import * as operations from "../../../core/operations/index";
+import { decapitalise } from "../../apiUtils";
+import excludedOperations from "../excludedOperations";
+
+const includedOperations = Object.keys(operations).filter((op => excludedOperations.indexOf(op) === -1));
+
+const dir = path.join(`${process.cwd()}/src/node`);
+if (!fs.existsSync(dir)) {
+    console.log("\nCWD: " + process.cwd());
+    console.log("Error: generateNodeIndex.mjs should be run from the project root");
+    console.log("Example> node --experimental-modules src/core/config/scripts/generateNodeIndex.mjs");
+    process.exit(1);
+}
+
+let code = `/**
+* THIS FILE IS AUTOMATICALLY GENERATED BY src/node/config/scripts/generateNodeIndex.mjs
+*
+* @author d98762625 [d98762625@gmail.com]
+* @copyright Crown Copyright ${new Date().getUTCFullYear()}
+* @license Apache-2.0
+*/
+
+/* eslint camelcase: 0 */
+
+
+import NodeDish from "./NodeDish";
+import { _wrap, help, bake, _explainExludedFunction } from "./api";
+import File from "./File";
+import {
+    // import as core_ to avoid name clashes after wrap.
+`;
+
+includedOperations.forEach((op) => {
+    // prepend with core_ to avoid name collision later.
+    code += `    ${op} as core_${op},\n`;
+});
+
+code +=`
+} from "../core/operations/index";
+
+global.File = File;
+
+// Define global environment functions
+global.ENVIRONMENT_IS_WORKER = function() {
+    return typeof importScripts === "function";
+};
+global.ENVIRONMENT_IS_NODE = function() {
```

Comment:
If we do end up moving these functions into Utils, they can be removed from here.